### ISSUE TITLE
Office Hours Name Disambiguation

### ISSUE THREAD

**Issue by prgoodwin** [unc-csxl/csxl.unc.edu#613]
As a professor, I am also using CSXL alongside the TA's and ULA's.  However, I request that the students start with a TA for homework help so that I can focus on professor related questions.  

In the CSXL system, it only shows as "Office Hours" and students can't tell which is mine and which are the TA's (which happen simultaneously).  It would be great if you could customize a title field that could disambiguate the different concurrent office hours.

**Comment by KrisJordan** [comment]
@jadekeegan I believe this could be incorporated into the push for recurring events #597 since we'll need a schema migration and this is a simple field added to events. @ajaygandecha thoughts?

**Comment by ajaygandecha** [comment]
@KrisJordan @jadekeegan Agree! This maybe can either be a Boolean field, or an enum dropdown. We can also add this to the event type by separating office hours to something like (Instructor OH | TA OH). Thoughts here?

**Comment by ItIsAndrewL** [comment]
@ajaygandecha This may be a good fix for the current conundrum I'm having with signage. My angle here is we could include a string field for "Office Hours Title" that defaults to the course name, but can be edited to fit the specific needs of the class (maybe this replaces the current `type` field, as I'm unsure what that does and can't get anything other than Office Hours to work).

**Comment by ajaygandecha** [comment]
@ItIsAndrewL I think this would be fine - we will just want to make sure that the UI is able to fit a title of arbitrary length. Looking like the office hours feature will be heavily expanded over the next few months.

The current hotfix pushed up (which also helps to aid this problem @prgoodwin!) is that the office hours event description is now shown onto students on the office hours card. For example, if the description says "Instructor OH", students can see this and help to differentiate between numerous events (merged in #675). We can expand this in the future, but Kris wanted to get this in quickly to aid with COMP110 and COMP210.

@ItIsAndrewL, you may be able to use this description, but there isn't any character limit yet.
